src/radical/entk/utils/prof_utils.py

```diff
@@ -219,42 +220,53 @@ def get_session_description(sid, src=None):
     return desc
 
 
-def write_workflow(workflow, uid, workflow_fout='entk_workflow', fwrite=True):
+# ------------------------------------------------------------------------------
+#
+def write_workflow(workflows, uid):
+def write_workflow(workflows, uid, workflow_fout='entk_workflow', fwrite=True):
```

Comment:
@andre-merzky did you happen to define twice the same function?

Reply:
Thanks - fixed!1